docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -590,6 +590,105 @@ Loading cert/key pair from "/var/lib/kubelet/pki/kubelet-client-current.pem
 "Failed to connect to apiserver" err="Get \"https://10.150.223.2:6443/healthz?timeout=1s\": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)"
 ```
 
+### Troubleshooting DNS issues in EKSA cluster
+
+DNS resolution is managed by coreDNS, a DNS server deployed as a pod within the cluster, it helps resolve service names to corresponding IP Addresses. If DNS resolution fails, applications maybe unable to connect to services or external endpoints.
+
+#### Common errors related to DNS issues:
+- Pods are unable to resolve the DNS names of other services.
+    ```
+    Error: Get http://example-service: no such host
+    ```
+- Delay in DNS resolution leading to timeouts and slow application responses.
+    ```
+    [ERROR] plugin/errors: 2 example-service.default.svc.cluster.local. A: read udp 10.0.0.2:59857->10.96.0.10:53: i/o timeout
+    ```
+
+#### DNS Debugging
+
+##### Verify CoreDNS deployment
+1. Check if CoreDNS pods are running
+    ```
+    kubectl get pods --namespace=kube-system -l k8s-app=kube-dns
+    ```
+    Expected Output:
+    ```
+    NAME                       READY     STATUS    RESTARTS   AGE
+    ...
+    coredns-7b96bf9f76-5hsxb   1/1       Running   0           1h
+    coredns-7b96bf9f76-mvmmt   1/1       Running   0           1h
+    ...
+    ```
+2. Check for errors in the DNS pod
+
+    ```
+    kubectl logs --namespace=kube-system -l k8s-app=kube-dns
+    ```
+    See if there are any suspicious or unexpected messages in the logs.
+3. Verify CoreDNS configuration and ensure it matches with cluster requirement, also verify if forward plugin points to correct DNS servers
```

Comment:
Its not clear what we want customers to verify here, or how they ensure that coredns is actually connected to the right nameserver

Reply:
We want customer to verify the nameserver by inspecting CoreDNS configuration in Corefile, it would either be defined in CoreFile as `forward . /etc/resolv.conf` or directly as `forward . 8.8.8.8`, to inspect `/etc/resolv.conf` customer or on-call support can use below steps to create debug pod and inspect `/etc/resolv.conf`, do you mean we should refer to below steps here in this section, to inspect `/etc/resolv.conf`?

> Verify CoreDNS configuration and ensure it matches with cluster requirement

In this point, what I meant is one should verify if the CoreDNS configMap has the correct nameserver which was set by admin/networking team.